config/credentials.yml.enc

```diff
@@ -1 +1 @@
-cTlXKCqJcBVl733P+/yb1LXqTgvV/y0D2LtFTePTIdV4J0G6p0BOAYrKLjUjaTvtDEMic6Yc6vEwHcAumAg3EYDfF4YbtDh5cigKIqm578zZwu8zQ38XAib3paWrCBaRcCi6NssjyMhF7xSrgP6uCYZ/t3kvFgYBNR6f+nHPt9/eWVyGkHPNobE2wiyzWjNYxzwLkxz55xt7Wa7Gr/vGCVNSZdlf0XL5qPt7oddqcdPaGwC8hKwPQ/3YLOlTsx/RMFy5ypaeRxM4Q9bVGO2iuqm4QePeb2OPSxzg+XgnBe9M71rA+0rYASNcdh9fx4AO+wbPQWu/RIoEIQF/gcoLWHOg2hCRwGPzFmuuO2WRsFMbDS7H1R/IheuEyI57ve2ZFfZ0MoizcSL/3LO8bLRJPc8F/EFRywmcqAWg--4PeiqEcd9kGoDtYl--xGfnseXkWayyt2FTUTBrRQ==
\ No newline at end of file
+XWOb4rEq4VTHwKy7BaInm+Vp9gkXkkpUJfz5douUIueJkjImwqEc+4g+1mDU1xJGYGPPdXLh+T8szevt8ReTjCeqZUqqEypgkPqsgVkGnTfAn69aVMK3Aj3mwGp/FsjS+4ZtcLcuEf5rizCNMSUfiFFC7TTLn/eIME9LR82mVOZNvRIEFsD/0CAq1iORi6Hr/1A7sM/pZmdyZY+WemMh7YWzrL1/anUAdZRAUzPYZpqJZhKATRukn8hvnmICzxCXSe4qxry0l7FFuH7gpR21Rbcdzd+1uJeZ7BRCcG5P6f9bUMar3AwBY32F2asLoYBJosdayKLagFYlB00e0Wd4DDWovVd9zb1LlkTdqZicP575Tu/QQNqJZq8Avic4kUpQkgDnw8bRPxzwDe3tzXwANQMr1UwwonGzYbf32Avf1TqDYj6xF+EpqK5sxIvQRoqRzQNeSrrR45nf0VjSdJuoISrQVvl8UqyMb/JywSMt3DkVuSB034ZwE8hwfAm9drVt22NY0hU8m1jfn19CHjrBGRXxN/bRB1Wf8xH7NpRTCA==--1JOnAEuLLhyxAheD--vocyfqHYC9OhayyxYcwNlQ==
```

Comment:
why have the credentials changed?
did you generate a new master.key?

Reply:
we put the stripe keys from thomas